presentation.md

```diff
@@ -63,14 +71,28 @@ Do głównych źródeł danych zaliczyć możemy:
 - IHME
 - koronawirusunas.pl
 - policja.pl
-
+:::
 
 # Techniki zbierania danych
 
-Główną techniką wykorzystywaną przez nas wył web scraping, stworzyliśmy skrypty,
+::: notes
+Główną techniką wykorzystywaną przez nas był web scraping, stworzyliśmy skrypty,
 które (oczywiście za zgodą właścicieli portali) pobierały z nich bieżące dane. Z
 kolei pozostałe dane były udostępnione publicznie w formie skompresowanych
 plików, także aktualizowanych codziennie.
+:::
+
+## Dostęp do API
+
+::: notes
+Część danych była dostępna bezpośrednio z API, tzn. wysyłając zapytanie do
+strony otrzymywaliśmy dane gotowe do przyjęcia przez program. Była to
+sytuacja o tyle prostsza od scrapingu, że nie musieliśmy dodatkowo przetwarzać
+tych danych.
+
+Wśród danych dostępnych przez API były dane pogodowe, jednak bardzo ułatwiało
+to pracę.
+:::
 
 ## Użycie API
```

Comment:
Przydałby się jakiś przykład do prezentacji. Postman, czy może jak to zrobić z pythona.

Reply:
API v1 już nie działa, więc napisałem o tym - nie jesteśmy w stanie pokazać tego na przykładzie meteostatu bez dodatkowej aktualizacji aktualizacji (#76)